src/common/addNew.js

```diff
@@ -0,0 +1,20 @@
+import { List } from './lists.js';
+
+const addNew = () => {
+  const list = new List();
+
+  const description = document.getElementById('input-todo').value;
+  list.add(description, false);
+  document.getElementById('input-todo').value = '';
+  const storedTodoList = JSON.parse(localStorage.getItem('toDolist'));
+  if (localStorage.getItem('toDolist') == null) {
+    localStorage.setItem('toDolist', JSON.stringify(list.list));
+    window.location.reload();
+  } else {
+    const newList = storedTodoList.concat(list.list);
+    localStorage.setItem('toDolist', JSON.stringify(newList));
+    window.location.reload();
+  }
+};
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5) or to be more clear, the 1st todo task in the todoList should have the index = 1, the second one(index = 2),...

![image](https://user-images.githubusercontent.com/29071984/185739202-18bea073-a556-4a4f-8f32-8661b04e937e.png)
As you can see from the above screenshot, the `index` of the ToDos remains `0`. Kindly have it updated as a new task is added

Reply:
Please check the screen shots. 
As the repo live link is displaying the indexes in my machine
<img width="1440" alt="Screenshot 2022-08-20 at 3 24 53 PM" src="https://user-images.githubusercontent.com/85638137/185740296-12c62887-257c-431f-9e43-2638e5a54242.png">
.